streaming_stt_nemo/__init__.py

```diff
@@ -12,6 +12,8 @@ class Model:
 
 
     def __init__(self, lang="en"):
+        # OVOS uses 'en-us' so this hacks that to work until dialects are supported
+        lang = lang.split('-')[0]
```

Comment:
This logic is OVOS dependent but this package is completely independent 

So please migrate this change to https://github.com/NeonGeckoCom/streaming-stt-nemo/pull/13